src/test/mocks/assets.ts

```diff
@@ -13,7 +13,7 @@ import merge from 'lodash/merge'
 
 export const rune: Asset = {
   chainId: ethChainId,
-  assetId: 'eip155:1/erc20:0x3155ba85d5f96b2d030a4966af206230e46849cb',
+  assetId: 'cosmos:thorchain-mainnet-v1/slip44:931',
```

Comment:
This isn't ERC20 RUNE

Reply:
[a408c11](https://github.com/shapeshift/web/pull/7098/commits/a408c1140029f13b71246fc2266679a6a3fcd5ba)